parm/atm/obs/config/sondes.yaml

```diff
@@ -0,0 +1,309 @@
+  #
+# - filter: Perform Action
+#   filter variables:
+#   - name: virtual_temperature
+#   action:
+#     name: assign error
+#     error function:
+#       name: ObsErrorModelStepwiseLinear@ObsFunction
+#       options:
+#         xvar:
+#           name: MetaData/air_pressure
+#         xvals: [100000, 95000, 90000, 85000, 35000, 30000, 25000, 20000, 15000, 10000, 7500, 5000, 4000, 3000, 2000, 1000]
+#         errors: [1.2, 1.1, 0.9, 0.8, 0.8, 0.9, 1.2, 1.2, 1.0, 0.8, 0.8, 0.9, 0.95, 1.0, 1.25, 1.5]
+  #
+  - filter: Perform Action
+    filter variables:
+    - name: surface_pressure
+    action:
+      name: assign error
+      error function:
+        name: ObsErrorModelStepwiseLinear@ObsFunction
+        options:
+          xvar:
+            name: ObsValue/surface_pressure
+          xvals: [80000, 75000]
+          errors: [110, 120]        # 1.1 mb below 800 mb and 1.2 mb agove 750 mb
+  #
+  - filter: Perform Action
+    filter variables:
+    - name: surface_pressure
+    action:
+      name: inflate error
+      inflation variable:
+        name: ObsErrorFactorSfcPressure@ObsFunction
+        options:
+          error_min: 100         # 1 mb
+          error_max: 300         # 3 mb
+  #
+  - filter: Perform Action
+    filter variables:
+    - name: specific_humidity
+    action:
+      name: assign error
+      error function:
+        name: ObsErrorModelStepwiseLinear@ObsFunction
+        options:
+          xvar:
+            name: MetaData/air_pressure
+          xvals: [25000, 20000, 10]
+          errors: [0.2, 0.4, 0.8]        # 20% RH up to 250 mb, then increased rapidly above
+          scale_factor_var: ObsValue/specific_humidity
+  #
+  - filter: Perform Action
+    filter variables:
+    - name: eastward_wind
+    - name: northward_wind
+    action:
+      name: assign error
+      error function:
+        name: ObsErrorModelStepwiseLinear@ObsFunction
+        options:
+          xvar:
+            name: MetaData/air_pressure
+          xvals: [100000, 95000, 80000, 65000, 60000, 55000, 50000, 45000, 40000, 35000, 30000, 25000, 20000, 15000, 10000]   #Pressure (Pa)
+          errors: [1.4, 1.5, 1.6, 1.8, 1.9, 2.0, 2.1, 2.3, 2.6, 2.8, 3.0, 3.2, 2.7, 2.4, 2.1]
+  #
+  # Reject when difference of wind direction is more than 50 degrees.
+  - filter: Bounds Check
+    filter variables:
+    - name: eastward_wind
+    - name: northward_wind
+    test variables:
+    - name: WindDirAngleDiff@ObsFunction
+      options:
+        minimum_uv: 3.5
+    maxvalue: 50.0
+    action:
+      name: reject
+    defer to post: true
+  # When multiple obs exist within a single vertical model level, inflate ObsError
+  - filter: Perform Action
+    filter variables:
+    - name: air_temperature
+    action:
+      name: inflate error
+      inflation variable:
+        name: ObsErrorFactorConventional@ObsFunction
+        options:
+          test QCflag: PreQC
+          inflate variables: [air_temperature]
```

Comment:
ufs_da tests add the line 
`pressure: MetaData/air_pressure
`
between PreQC and inflate.   Not sure if this is still needed.   This comment applies to other PreQC sections in this yaml.

Reply:
Yes, I will add this one and the other in a following commit.

Reply:
Done.